### PR TITLE
ShowPicture: Support empty names

### PR DESCRIPTION
**Pull request by Ghabry**
Yume2kki 0.116 started to show a picture with no filename on the PC-wallpaper map.

No idea why they are doing this but when this is not filtered it unloads a black picture that is used for a background effect.

See #2519

a

@purupurupuruin

### CODE REVIEW

**Comment by purupurupuruin** [comment]
I tried it but something weird happens, when I select a wallaper to preview it and cancel out it the background effect doesn't come back.

**Comment by Ghabry** [comment]
You are right :(
So this is not the only issue.

**Comment by fdelapena** [comment]
Not sure if related to the bug mentioned below and RPG_RT version used:

https://github.com/EasyRPG/Player/blob/602b229d311d31c32c897ddaed7482d3ff7bc0c1/src/game_pictures.cpp#L265-L269

**Comment by Ghabry** [comment]
Looked into RPG_RT to figure this out:

It will apply all the Picture command data as usual but when there is no string the image buffer is not overwritten.

So this reuses the image buffer of whatever was in that slot before.

**Comment by Ghabry** [comment]
There were multiple minor differences to RPG_RT that made this incompatible:

1. ShowPicture in RPG_RT will apply all the parameters but ignore the image buffer when the filename is empty
- Player hid the image and then did nothing
- I still hide the image for the "file has name case" due to Async loading (is made visible right after Async loading as before)

2. RPG_RT will render images with empty names
- Player aborted the rendering here

3. ErasePicture in RPG_RT will unload the image (and allocate a new image with 0 bytes, but this doesn't matter)
- Player only hid the sprite. This is actually bad because this means the reference is never lost until the picture is replaced -> waste of memory

With all these behaviours matching now the effect in the wallpaper select works.

File: src/output.cpp [hunk 1]
```diff
@@ -341,7 +341,7 @@ void Output::ErrorStr(std::string const& err) {
 		std::cout << "EasyRPG Player will close now.";
 #if defined (GEKKO) || defined(__SWITCH__) || defined(_3DS)
 		// stdin is non-blocking
-		sleep(5);
+		Game_Clock::SleepFor(5s);
```
Comment (carstene1ns) [inline src/output.cpp:344]: I guess <unistd.h> might be removed for switch at least, wii likely.

Comment (Ghabry) [reply src/output.cpp:339]: ok, also tidyed the file a bit

**Comment by purupurupuruin** [comment]
Tested it, works perfectly as far as I can tell!

**Review by carstene1ns — APPROVED** [review]

**Review by fdelapena — APPROVED** [review]

**Pull request merged**